From: Dean Nelson <dnelson@redhat.com>
Date: Wed, 10 Mar 2010 18:53:48 -0500
Subject: [hwmon] add 0x prefix to hex coretemp module output
Message-id: <20100310185347.5435.20899.send-patch@aqua>
Patchwork-id: 23540
O-Subject: [RHEL5.6 PATCH] hwmon: add hex '0x' indication to coretemp module
	output
Bugzilla: 571864
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Jarod Wilson <jarod@redhat.com>
RH-Acked-by: John Feeney <jfeeney@redhat.com>

Resolves RHBZ 571864.

Avoid hex and decimal confusion when printing out the cpu model.

Backport of a patch just recently submitted by prarit to lm-sensors:
http://lists.lm-sensors.org/pipermail/lm-sensors/2010-March/028011.html

Tested successfully on dell-per610-01.lab.bos.redhat.com by me.


diff --git a/drivers/hwmon/coretemp.c b/drivers/hwmon/coretemp.c
index 6d54c8c..28b28e9 100644
--- a/drivers/hwmon/coretemp.c
+++ b/drivers/hwmon/coretemp.c
@@ -379,7 +379,7 @@ static int __init coretemp_init(void)
 			   family 6 CPU */
 			if ((c->x86 == 0x6) && (c->x86_model > 0xf))
 				printk(KERN_WARNING DRVNAME ": Unknown CPU "
-					"model %x\n", c->x86_model);
+					"model 0x%x\n", c->x86_model);
 			continue;
 		}
 
